plugins/experimental/webp_transform/ImageTransform.cc

```diff
@@ -183,7 +183,7 @@ class GlobalHookPlugin : public GlobalPlugin
 
     // If we might need to convert check to see if what the browser supports
     if (transaction_convert_to_webp == true || transaction_convert_to_jpeg == true) {
-      std::string accept  = transaction.getServerRequest().getHeaders().values("Accept");
+      std::string accept  = transaction.getClientRequest().getHeaders().values("Accept");
```

Comment:
The next few lines of code decide whether and how to transform the body of the response to client to a different content type.  So, it make more sense to look at what content types the client will accept, not what types we told the server ATS would accept.

Reply:
True, if there is not another transform plugin, which supposedly modified Accept header, between this plugin and the client.